src/AsmResolver/Utf8String.cs

```diff
@@ -19,6 +19,10 @@ public sealed class Utf8String :
         IEquatable<byte[]>,
         IComparable<Utf8String>,
         IEnumerable<char>
+#if NET10_0_OR_GREATER
+        , ISpanFormattable
+        , IUtf8SpanFormattable
+#endif
```

Comment:
`ISpanFormattable` is already available in .NET 6.0 ([msdn](https://learn.microsoft.com/en-us/dotnet/api/system.ispanformattable?view=net-10.0)) and `IUtf8SpanFormattable` in .NET 8 ([msdn](https://learn.microsoft.com/en-us/dotnet/api/system.iutf8spanformattable?view=net-9.0))

Reply:
We have to care because some users of AsmResolver also depend on dynamic features of the runtime (e.g., loading assemblies compiled for older runtimes with reflection alongside AsmResolver)

Reply:
Ah, gotcha. Fixed! 🙂